cli/src/scanner.rs

```diff
@@ -69,16 +69,21 @@ impl FileSet {
                 return Ok::<(), anyhow::Error>(());
             }
 
-            let original_path = path
+            let original_path_abs = path
+                .to_str()
+                .expect("failed to convert path to string")
+                .to_string();
+            let original_path_rel = path
+                .strip_prefix(repo_root)
+                .unwrap_or(&path)
                 .to_str()
                 .expect("failed to convert path to string")
                 .to_string();
```

Comment:
I know you're following the already existing pattern here, but I think we need to update this so it doesn't panic here. Instead of using `expect` we can use `map_err` + `?` (try sigil)

Reply:
Thanks! went with `ok_or_else` since I was working with an `Option` rather than a `Result`